aws/services/iam/resource.ftl

```diff
@@ -105,6 +106,12 @@
         ]
     [/#list]
 
+    [#local constructedName=""]
+    [#if ! name?has_content]
+        [#local constructedName=name]
+    [/#if]
```

Comment:
Do we have the constructed name anymore? if not I think this can be removed and a check for a name parameter would be used?

Reply:
True, done